<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>170</width>
    <height>600</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>170</width>
    <height>600</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>800</width>
    <height>600</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Chisto</string>
  </property>
  <widget class="QListWidget" name="listWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>50</y>
     <width>150</width>
     <height>540</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>150</width>
     <height>540</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>150</width>
     <height>540</height>
    </size>
   </property>
  </widget>
  <widget class="QTextEdit" name="textEdit">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>220</y>
     <width>620</width>
     <height>370</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>620</width>
     <height>370</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>620</width>
     <height>370</height>
    </size>
   </property>
  </widget>
  <widget class="QFrame" name="frame_ats">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>600</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>800</width>
     <height>600</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>800</width>
     <height>600</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(39, 206, 225);</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
  </widget>
  <widget class="QFrame" name="frame_gw">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>600</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>800</width>
     <height>600</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>800</width>
     <height>600</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 217, 79, 255);</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_gw">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>5</y>
     <width>50</width>
     <height>40</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>50</width>
     <height>40</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>50</width>
     <height>40</height>
    </size>
   </property>
   <property name="text">
    <string>GW</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_ats">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>5</y>
     <width>50</width>
     <height>40</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>50</width>
     <height>40</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>50</width>
     <height>40</height>
    </size>
   </property>
   <property name="text">
    <string>ATS</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_ip">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>10</y>
     <width>90</width>
     <height>20</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>90</width>
     <height>20</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>90</width>
     <height>20</height>
    </size>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
    </font>
   </property>
   <property name="text">
    <string>IP address</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <zorder>frame_ats</zorder>
  <zorder>frame_gw</zorder>
  <zorder>listWidget</zorder>
  <zorder>textEdit</zorder>
  <zorder>btn_gw</zorder>
  <zorder>btn_ats</zorder>
  <zorder>label_ip</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
